content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -83,21 +91,24 @@ Pick a control plane node that you wish to upgrade first. It must have the `/etc
 
   {{< tabs name="k8s_install_kubeadm_first_cp" >}}
   {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+
   ```shell
-   # replace x in {{< skew currentVersion >}}.x-00 with the latest patch version
-   apt-mark unhold kubeadm && \
-   apt-get update && apt-get install -y kubeadm={{< skew currentVersion >}}.x-00 && \
-   apt-mark hold kubeadm
-   ```
+  # replace x in {{< skew currentVersion >}}.x-00 with the latest patch version
+  apt-mark unhold kubeadm && \
+  apt-get update && apt-get install -y kubeadm={{< skew currentVersion >}}.x-00 && \
+  apt-mark hold kubeadm
+  ```
+
   {{% /tab %}}
   {{% tab name="CentOS, RHEL or Fedora" %}}
+
   ```shell
   # replace x in {{< skew currentVersion >}}.x-0 with the latest patch version
   yum install -y kubeadm-{{< skew currentVersion >}}.x-0 --disableexcludes=kubernetes
   ```
+
   {{% /tab %}}
   {{< /tabs >}}
-  <br />
 
 - Verify that the download works and has the expected version:
```

Comment:
These should be numbered steps, not bullets

Reply:
Thanks. Have changed to numbered steps now. Not sure correct or not now.